scripts/reset_localization.rb

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env ruby
+# frozen_string_literal: true
+
+require 'os'
+require_relative '../RubySetupSystem/RubyCommon'
+
+def editor
+  return 'poedit' unless which('poedit').nil?
+
+  core_editor = `git config --global core.editor`
+  return core_editor.strip unless core_editor.empty?
+
+  visual_editor = `git config --global core.visual`
+  return visual_editor.strip unless visual_editor.empty?
+
+  if OS.windows?
```

Comment:
seems like `OS.win?` breaks on windows?? It complained that it was not found, but on linux it worked? Anyways, this works.

Reply:
I see, I misremembered the name of the thing to check, it indeed is `windows?`